labs/RivePlayer/src/AnimationTimer.cs

```diff
@@ -54,12 +54,29 @@ private async void InvalLoopAsync(int continuationToken)
     {
         while (continuationToken == _invalLoopContinuationToken)
```

Comment:
@csmartdalton - @Sergio0694 suggested we use a `DispatcherQueueTimer` here for better performance, you can see a similar scenario we have in a Toolkit control here: https://github.com/CommunityToolkit/WindowsCommunityToolkit/blob/2233ac655ff24bc77a9fa23f5f70b0434230f342/Microsoft.Toolkit.Uwp.UI.Controls.Core/RotatorTile/RotatorTile.cs#L389-L394

Basically the `Tick` event handler becomes our new method below and you can get rid of most of the other code here in your class. 🙂 Let us know if you need a hand.

Reply:
I tried using `DispatcherQueueTimer` but it looks like it's not available everywhere:

![notavailable](https://user-images.githubusercontent.com/99840794/206110278-77e31318-78a9-4fec-b1c5-423a7a3aea22.png)

Is there a way to use it in net6.0 or would it only be a partial solution?

Reply:
@csmartdalton it's available, it's just a different namespace/mapping which should be handled by Labs, so you should be able to ignore the VS warning (I'm not sure why it doesn't understand that probably as it's looking for the fully qualified type name or something).

Filed an issue with the docs here: https://github.com/MicrosoftDocs/winrt-api/issues/2309

Reply:
Filed https://github.com/CommunityToolkit/Labs-Windows/issues/331 for tracking.

Reply:
Thanks for the offline help @michael-hawker ! It's working now.